Tchap/Managers/Registration/RegistrationService.swift

```diff
@@ -44,32 +44,59 @@ final class RegistrationService: RegistrationServiceType {
     
     // MARK: - Public
     
-    func submitRegistrationEmailVerification(to email: String, completion: @escaping (MXResponse<ThreePIDCredentials>) -> Void) {
-        self.submitRegistrationEmailVerification(to: email, using: self.restClient, completion: completion)
+    func initRegistrationSession(completion: @escaping (MXResponse<String>) -> Void) {
```

Comment:
Preferable to use `setup` prefix to avoid confusion with `init` methods.

Reply:
ok